vine/src/ast.rs

```diff
@@ -584,6 +590,13 @@ pub enum ComparisonOp {
   Ge,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum Bound {
```

Comment:
Since this isn't used outside of the `Builtins` enum, I think this should be inlined there (so there would be `Builtins::BoundUnbounded`,` `Builtins::BoundInclusive`, `Builtins::BoundExclusive`)